### DISCUSSION TITLE
How to get nested flashMessages error?

### DISCUSSION THREAD

**Discussion by galaczi** [adonisjs/core#3805] [Help] [answered]
I have a client side generated form where a user can create new rows. The values with the same input ``name`` are put in an array by Adonis. During validation the error messages are reported by adding the index.

```
 "errors": {
    "'subject.2'": [
      'required validation failed'
    ]
  }
```
How can I get or check this message? ``flashMessages.get('errors.subject.2')`` returns ``undefined``.

Edit: ``{{flashMessages.get('errors["subject.2"]')}}`` works but I am interested in something like ``flashMessages.has('errors.subject.*')`` to check if any row has errors.

**Comment by Barbapapazes** [comment, marked as answer]
You can use the object returned by the get method!

So `flashMessages.get('errors')['subject.2']`should works!
